README.md

```diff
@@ -142,8 +142,7 @@ For, Lisk Sepolia Testnet:
     --rollup.halt=major \
     --port=30303 \
     --rollup.disabletxpoolgossip=true \
-    --override.canyon=0 \
-    --override.fjord=1716998400
+    --override.canyon=0
```

Comment:
we could remove this as well?

Reply:
This is still necessary because the [genesis config](https://raw.githubusercontent.com/LiskHQ/lisk-node/36d515946080bc059276ae42d7d64de9fbb71ff1/sepolia/genesis.json#L15270-L15289) for Lisk Sepolia still doesn't specify `canyonTime` unlike Lisk mainnet. It exists only in the rollup.json, but that already existed and we were asked to specify the flag specifically.